tests/destructor/t11833.nim

```diff
@@ -0,0 +1,12 @@
+discard """
```

Comment:
I added this test to some existing test, so please remove it.

Reply:
Ok